egs/aspire/s5/run.sh

```diff
@@ -9,6 +9,7 @@
 . ./path.sh
 
 mfccdir=`pwd`/mfcc
+aspire_data=/export/corpora/LDC/LDC2017S21/IARPA-ASpIRE-Dev-Sets-v2.0/data
```

Comment:
Better to check for the cluster and set the variable e.g. see AMI script ([line](https://github.com/kaldi-asr/kaldi/blob/master/egs/ami/s5b/run.sh#L30))